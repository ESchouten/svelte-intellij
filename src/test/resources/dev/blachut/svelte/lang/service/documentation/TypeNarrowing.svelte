<script lang="ts">
  /**
   * docs have to be removed!
   */
  let local = 323 as number | string;
</script>

{#if typeof local === "number"}
  {<caret>local.toFixed()}
{/if}
