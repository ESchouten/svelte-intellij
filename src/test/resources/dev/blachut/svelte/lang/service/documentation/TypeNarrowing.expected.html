let local: number<br/>[ide]<br/>let local: number | string